flink-kubernetes-operator/src/main/java/org/apache/flink/kubernetes/operator/utils/SavepointUtils.java

```diff
@@ -0,0 +1,43 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.flink.kubernetes.operator.utils;
+
+import org.apache.flink.kubernetes.operator.crd.FlinkDeployment;
+
+/** Savepoint utilities. */
+public class SavepointUtils {
+
+    public static final String SAVEPOINT_ANNOTATION_KEY =
+            "flinkdeployments.flink.apache.org/savepoint";
+    public static final String SAVEPOINT_ANNOTATION_VALUE = "trigger";
+
+    public static boolean checkSavepointAnnotation(FlinkDeployment flinkDeployment) {
+        return flinkDeployment.getMetadata().getAnnotations() != null
+                && SAVEPOINT_ANNOTATION_VALUE.equals(
```

Comment:
Could the value of the `SAVEPOINT_ANNOTATION_VALUE` be a boolean value which is user-friendly and no need to be a specified value？

Reply:
To my best knowledge k8s annotations are key-value pairs.

Reply:
@morhidi, I mean that the `SAVEPOINT_ANNOTATION_KEY` could be `flinkdeployments.flink.apache.org/trigger-savepoint` and `SAVEPOINT_ANNOTATION_VALUE` could be `true`.

Reply:
We can do that, yeah.